AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/ItineraryController.cs

```diff
@@ -128,7 +127,7 @@ public ItineraryController(IMediator mediator, IItineraryEditModelValidator itin
         [Route("Admin/Itinerary/{id}/[Action]")]
         public async Task<IActionResult> SelectRequests(int id)
         {
-            var orgId = await _mediator.SendAsync(new OrganizationIdQuery { ItineraryId = id });
+            var orgId = await GetOrganizationIdBy(id);
```

Comment:
@tonysurma PR #892. It should NOT be merged yet, b/c I have more work to do... unless you want multiple PR's against the Issue #874. I just pushed some .ConfigureAwait(false) changes to this PR that sean feldman found below.

Let me know if it's easier for you to have it broken up into separate PR's for this issue